### DISCUSSION TITLE
You do not have a valid subscription for this server.

### DISCUSSION THREAD

**Discussion by holland-built** [tteck/Proxmox#2544] [Help Needed]
New install and after running the post install it still shows "You do not have a valid subscription for this server."    Never had this issue.  Any logs I can provide to help resolve.

**Comment by tteck** [comment]
clear cache

**Comment by holland-built** [comment]
thank you for the fast response.   that didnt help I can give you access or screen shots or whatever

**Reply by tteck** [reply]
All good now?

**Comment by tteck** [comment]
Show the output from `grep "NoMoreNagging" /usr/share/javascript/proxmox-widget-toolkit/proxmoxlib.js`

**Comment by holland-built** [comment]
grep "NoMoreNagging" /usr/share/javascript/proxmox-widget-toolkit/proxmoxlib.js
                        .data.status.toLowerCase() == 'NoMoreNagging') {

**Reply by tteck** [reply]
It is installed. It's a cache issue. Log out of Proxmox, at the login screen, do a `ctrl+r` then login

**Comment by holland-built** [comment]
no its still there but I wont worry about it.  Just cosmetic. here is a video 
https://youtu.be/HY0NJk6DXE0
